STYLE.rst

```diff
@@ -235,10 +235,6 @@ blank line. Pragmas should be ordered alphabetically. Example:
     {-# LANGUAGE FlexibleInstances #-}
     {-# LANGUAGE QuasiQuotes #-}
 
-    #if __GLASGOW_HASKELL__ < 806
-    {-# LANGUAGE TypeInType #-}
-    #endif
-
```

Comment:
I didn't even think to check the style guide (said every clash developer in unison)

Reply:
Me neither.
But my global-find did.

Reply:
My ~only~ mistake was searching only `clash-{prelude,lib,ghc}`